src/expression.c

```diff
@@ -947,7 +947,7 @@ Expression *resolveUFCSProperties(Scope *sc, Expression *e1, Expression *e2 = NU
  * Perform semantic() on an array of Expressions.
  */
 
-bool arrayExpressionSemantic(Expressions *exps, Scope *sc)
+bool arrayExpressionSemantic(Expressions *exps, Scope *sc, bool preserveErrors)
```

Comment:
Can `preserveErrors` be made an enum? The meaning of `true`/`false` arguments isn't obvious.